doc/changes/latest.inc

```diff
@@ -80,6 +80,8 @@ Bugs
 
 - Fix bug with :ref:`mne make_scalp_surfaces` where ``--overwrite`` was not functional (:gh:`8800` by `Yu-Han Luo`_)
 
+- Fix bugs with `mne.io.read_raw_persyst` where multiple ``Comments`` with the same name are allowed, and ``Comments`` with a "," character are now allowed (:gh:`8311` and :gh:`8806` by `Adam Li`_ and `Andres Rodriguez`_)
```

Comment:
`Andres Rodriguez` is a new contributor (congrats!), so this should go to the top of the section and have new-contributor highlighting on the name